staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go

```diff
@@ -152,6 +153,7 @@ func (flags *WaitFlags) AddFlags(cmd *cobra.Command) {
 
 	cmd.Flags().DurationVar(&flags.Timeout, "timeout", flags.Timeout, "The length of time to wait before giving up.  Zero means check once and don't wait, negative means wait for a week.")
 	cmd.Flags().StringVar(&flags.ForCondition, "for", flags.ForCondition, "The condition to wait on: [delete|condition=condition-name[=condition-value]|jsonpath='{JSONPath expression}'=[JSONPath value]]. The default condition-value is true.  Condition values are compared after Unicode simple case folding, which is a more general form of case-insensitivity.")
+	cmd.Flags().BoolVar(&flags.WaitForCreation, "wait-for-creation", flags.WaitForCreation, "Default behavior of wait command errors out when the resource does not exist. If this flag is specified, command also waits the creation of the resource, in case it doesn't exist.")
```

Comment:
An example in the `--help` output showing how to use this would be helpful.

Reply:
Added.

Reply:
Added.